Apps/Inc/Tasks.h

```diff
@@ -134,36 +128,60 @@ typedef enum{
     OS_MAIN_LOC = 0x200,
     OS_CANDRIVER_LOC = 0x400,
     OS_MOTOR_CONNECTION_LOC = 0x800,
-    OS_DISPLAY_LOC = 0x1000
+    OS_DISPLAY_LOC = 0x1000,
+    OS_TASKS_LOC = 0x2000,
 } os_error_loc_t;
```

Comment:
@IshDeshpa thought the plan was to get rid of LOC, is that a diff PR/issue ticket